status-im/status-react
# chore: bump status-go in order to make use of new peermgmt

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
Bump up status-go version to use latest peer-mgmt changes implemented in go-waku. 
This would reduce or control the number of relay peer connections to 16 and maximum peer connections to 20. This should help in reducing bandwidth usage and also help protect app  in case it gets flooded with incoming peer connections.

Note: Would like to know how the app's bandwidth usage is being tracked to know if this change improves it.

### Testing notes
This has already been tested with status-desktop locally to ensure minimum required peer connections are available.
This change will be dogfooded with help of some internal Waku CC's to ensure there are no side effects.

#### Platforms

- Android
- iOS


#### Areas that maybe impacted


##### Functional

- networks
- mailservers

##### Non-functional

- battery performance - May improve
- CPU performance / speed of the app - May improve as less data to be processed if number of peers are limited. 
- network consumption - should reduce as number of peers where connection would be initiated to will get limited to 10 and overall peer connections would be limited to 20.

### Steps to test


- Install this version of mobile app
- Go to Settings -> Advanced -> Peers stats 
- Ensure that peers count shown never grows beyond 20.
- Would be great if bandwidth usage can also be recorded. 
- Verify by sending and receiving messages using a test community.



status: wip
